recipes/krb5/bld.bat

```diff
@@ -0,0 +1,16 @@
+:: First, make sure you have sed, gawk, cat, and cp
+PATH C:\msys64\%MSYSTEM%\bin;C:\msys64\usr\bin;%PATH%
```

Comment:
It turns out that this was all I needed to get `sed, gawk, cat, and cp` from `MSYS2`. Not sure if that is the way to go though.